broker-cli/commands/wallet.js

```diff
@@ -46,6 +46,72 @@ const SUPPORTED_COMMANDS = Object.freeze({
   UNLOCK: 'unlock'
 })
 
+function balancesTable (balances, logger) {
+  const balancesTable = new Table({
+    head: ['', 'Committed (Pending)', 'Uncommitted (Pending)'],
+    style: { head: ['gray'] }
+  })
+
+  balances.forEach((balance) => {
+    let {
+      symbol,
+      error = null,
+      totalChannelBalance,
+      totalPendingChannelBalance,
+      uncommittedBalance,
+      uncommittedPendingBalance
+    } = balance
+
+    totalChannelBalance = error ? 'Not Available'.yellow : totalChannelBalance.green
+    uncommittedBalance = error ? 'Not Available'.yellow : uncommittedBalance
+
+    // We fix all pending balances to 8 decimal places due to aesthetics. Since
+    // this balance should only be temporary, we do not care as much about precision
+    totalPendingChannelBalance = error ? '' : `(${Big(totalPendingChannelBalance).toFixed(8)})`.grey
+    uncommittedPendingBalance = error ? '' : `(${Big(uncommittedPendingBalance).toFixed(8)})`.grey
+
+    balancesTable.push([
+      symbol,
+      `${totalChannelBalance} ${totalPendingChannelBalance}`,
+      `${uncommittedBalance} ${uncommittedPendingBalance}`
+    ])
+  })
+
+  logger.info('Wallet Balances'.bold.white)
+  logger.info(balancesTable.toString())
+}
+
+function balancesJSON (balances, logger) {
+  const balancesArray = balances.map((balance) => {
+    let {
+      symbol,
+      error = null,
+      totalChannelBalance,
+      totalPendingChannelBalance,
+      uncommittedBalance,
+      uncommittedPendingBalance
+    } = balance
+
+    totalChannelBalance = error ? 'Not Available' : totalChannelBalance
+    uncommittedBalance = error ? 'Not Available' : uncommittedBalance
+
+    // We fix all pending balances to 8 decimal places due to aesthetics. Since
+    // this balance should only be temporary, we do not care as much about precision
+    totalPendingChannelBalance = error ? '' : `(${Big(totalPendingChannelBalance).toFixed(8)})`
+    uncommittedPendingBalance = error ? '' : `(${Big(uncommittedPendingBalance).toFixed(8)})`
+
+    return {
+      symbol: symbol,
+      totalChannelBalance: totalChannelBalance,
+      totalPendingChannelBalance: totalPendingChannelBalance,
+      uncommittedBalance: uncommittedBalance,
+      uncommittedPendingBalance: uncommittedPendingBalance
+    }
+  })
+
+  logger.info({ 'walletBalances': balancesArray })
```

Comment:
why is `walletBalances` in quotes?
Also, how did you choose `walletBalances` as the name here?